<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfaceClient</class>
 <widget class="QDialog" name="interfaceClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>19</y>
     <width>931</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="accueil"/>
   <widget class="QWidget" name="client">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1171</width>
       <height>521</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter client</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Code client :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_nom">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_prenom">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Né(e) le :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_adresse">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>N° de téléphone  :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_num_tel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>E-mail :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_email">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type du permis :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_permis_type">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>310</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Permis n° :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_permis_numero">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Délivré le :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fin de validité :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pièce ID n° :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_cin">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>400</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_quitter">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_code2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="table_client">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>521</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_47">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Code du client :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer </string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tridesc">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>420</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri Desc</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_afficher">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>420</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimer">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="client_date_naissance">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="client_date_del">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>340</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="client_datefin_del">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>370</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="code">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="convention">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1171</width>
       <height>521</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter convention</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Code convention :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="c_nom">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="c_adresse">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>N° de téléphone  :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="c_numtel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>E-mail :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="c_email">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_quitter_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="c_code2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="table_client_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>521</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_48">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Code du client :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer </string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher_2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tridesc_2">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>420</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri Desc</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_afficher_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>420</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimer_2">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="c_code">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="email">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>771</width>
       <height>441</height>
      </rect>
     </property>
     <property name="title">
      <string>Envoyer un E-mail</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>671</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="4" column="0">
        <widget class="QLabel" name="label_27">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;</string>
         </property>
         <property name="text">
          <string>Destinataire :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="port">
         <property name="font">
          <font>
           <family>Microsoft YaHei Light</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;</string>
         </property>
         <property name="text">
          <string>465</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_35">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;
</string>
         </property>
         <property name="text">
          <string>Message :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="server">
         <property name="font">
          <font>
           <family>Microsoft JhengHei Light</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;</string>
         </property>
         <property name="text">
          <string>smtp.gmail.com</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_40">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;</string>
         </property>
         <property name="text">
          <string>Server port :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="uname">
         <property name="font">
          <font>
           <family>Microsoft YaHei Light</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPlainTextEdit" name="msg">
         <property name="font">
          <font>
           <family>Microsoft YaHei Light</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_37">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;</string>
         </property>
         <property name="text">
          <string>Mot de passe :</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_39">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;</string>
         </property>
         <property name="text">
          <string>Objet :</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="sendBtn">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Envoyer</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_mail">
         <property name="font">
          <font>
           <family>Microsoft YaHei Light</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="subject">
         <property name="font">
          <font>
           <family>Microsoft YaHei Light</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="paswd">
         <property name="font">
          <font>
           <family>Microsoft YaHei Light</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_38">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:none;</string>
         </property>
         <property name="text">
          <string>Nom d'utilisateur :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_36">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Smtp-server :</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
